frontend/src/model/beta/settings/ModelAccess.tsx

```diff
@@ -58,11 +60,13 @@ export default function ModelAccess({ model }: ModelAccessProps) {
   }
 
   async function updateAccessList() {
+    setLoading(true)
     const res = await patchModel(model.id, { collaborators: accessList })
     if (!res.ok) {
+      setLoading(false)
```

Comment:
This can be removed. We're already doing this on line 80 (which is the correct place to do it). This one is in the wrong place and adding no value.